common/functions.sh

```diff
@@ -33,7 +33,7 @@ get_collections_list() {
 get_scl_namespace() {
   el_version=${2-`os_major_version`}
   for namespace in rh sclo ; do
-    grep -e "^[[:space:]]*$1[[:space:]]*$" "`dirname ${BASH_SOURCE[0]}`"/../PackageLists/collections-list-$namespace-el$el_version >/dev/null
+    grep -e "$1" "`dirname ${BASH_SOURCE[0]}`"/../PackageLists/collections-list-$namespace-el$el_version >/dev/null
```

Comment:
Would `"^[[:space:]]*$1[[:space:]]*"` work instead of `"$1"`? I think about a potential case where we would have `mysql55` in one namespace and e.g. `sclo-mysql55` in another namespace..

Reply:
Ah, so that is why the `[[:space:]]`s were there…

Dropping the End-of-Line anchor (`$`) should work too. I will change it to more closely match the original version, and add an explanation comment.